Sourcery/Generating/Generator.swift

```diff
@@ -101,10 +93,39 @@ extension Type {
 }
 
 enum Generator {
+    private static func updateTypeRelationship(for type: Type, typesByName: [String: Type], processed: inout [String: Bool]) {
+        type.based.keys.forEach { name in
+            guard let baseType = typesByName[name] else { return }
```

Comment:
I think that's configured already, it's iteration over based after all so that's a no op?